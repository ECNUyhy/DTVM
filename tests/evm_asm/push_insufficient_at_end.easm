PUSH1 0x11
PUSH1 0x22
ADD
HEX 0x621234
// `HEX` is not an actual EVM opcode;
// This testcase only used to simulate test scenarios where there are insufficient data bytes.
