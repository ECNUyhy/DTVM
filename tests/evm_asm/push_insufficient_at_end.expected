status: success
error_code: 0
stack:
  - '1234'
  - '33'
memory: ''
storage: {}
transient_storage: {}
return: ''
events: []
